### PR TITLE
Improve GMT dataset

### LINKED ISSUE #362: 修改GMT数据概述？
我们是否可以像 [中国地理空间数据集](https://docs.gmt-china.org/latest/dataset-CN/) 那样在 index 简要介绍GMT数据。即把 [GMT数据概述](https://docs.gmt-china.org/latest/dataset/overview/#) 中每小节总结的话移动 index 中。概述只包括技术细节，要不然每类数据总结隐藏的太深。我现在看index，其实不知道每种数据到底是啥类型。

> GMT内置数据是指在安装GMT时就已经安装了的数据，因而可以直接使用。这类数据包括
> - GSHHG: 全球高分辨率海岸线数据
> - DCW: 世界数字图表

> GMT远程数据保存在GMT数据服务器上。当需要使用某个数据时，GMT会自动将数据下载到本地。目前，GMT提供了如下远程数据：
> - earth_relief: 全球地形起伏数据
> - earth_day 和 earth_night: 地球昼夜卫星影像
> - earth_age: 地球洋壳年龄数据
> - earth_mask: 地球掩膜数据

> 除了GMT官方提供的内置数据和远程数据外，用户还可以自行准备数据，建立GMT数据库，供GMT使用。GMT中文社区整理了一些自定义数据，用户可以选用，包括：
> - GADM: 全球行政区划数据库
> - PB2002: 全球板块边界数据
> - 中国地理空间数据集

### PR DESCRIPTION
**Pull request by core-man**
- `overview` 中每种数据的简介部分移动到 `index` 中
- `overview` 改名为 `usage`，主要包括数据的使用方法
- 远程数据的[工作原理及技术细节](https://docs.gmt-china.org/latest/dataset/overview/#id3)暂时保留在 `usage` 中，是否需要移动到 index? 我觉得index 最好简短一点，因此 no。
Fixes #362.

### CODE REVIEW

**Comment by core-man** [comment]
This is a quick commit. Please see how it looks before we further polish it. Ping @CovMat @wangliang1989 @seisman for comments.

**Comment by seisman** [comment]
Looks great! Anything else you want to do in this PR?

**Comment by core-man** [comment]
> Looks great! Anything else you want to do in this PR?

I'd like to polish the index a little more, while additional PRs will be submitted to polish othet pages.

**Comment by seisman** [comment]
> > Looks great! Anything else you want to do in this PR?
> 
> I'd like to polish the index a little more (today), while additional PRs will be submitted to polish othet pages.

Sounds good. Please keep the changes small to make the PR easy to review.

File: source/dataset/index.rst [hunk 1]
```diff
@@ -1,15 +1,58 @@
...
+
+使用方法
+--------
+
+这三类数据的使用方法都比较简单：
+
+.. toctree::
+   :maxdepth: 2
+
+   usage
```
Comment (core-man) [inline source/dataset/index.rst:19]: This structure still looks strange.

Comment (seisman) [reply source/dataset/index.rst:19]: Can we just hide the "usage" section?

Comment (core-man) [reply source/dataset/index.rst]: done

File: source/dataset/index.rst [hunk 2]
```diff
@@ -8,16 +8,6 @@
...
 
-使用方法
---------
-
-这三类数据的使用方法都比较简单：
-
-.. toctree::
-   :maxdepth: 2
-
-   usage
```
Comment (seisman) [inline source/dataset/index.rst]: 删除后会导致整个 usage 一节消失。改成：

```
.. toctree::   
   :maxdepth:
   :hidden:
 
   usage
```

Comment (core-man) [reply source/dataset/index.rst]: fixed

**Review by seisman — APPROVED** [review]

**Pull request merged**